### PR TITLE
List pod images by app

### PR DESCRIPTION
**Pull request by pkaramol**

### CODE REVIEW

File: specs/kubernetes/list_pod_images_by_app.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+command: >
+  kubectl get pods -l app={{app}} -o jsonpath='{range .items[*]}{"\n"}{range .spec.containers[*]}{.name}{"\t"}{.image}{"\n"}{end}{end}' | sort | uniq
+tags:
+  - kubernetes
+description: Lists the images of pods fetched by app label
+arguments:
+  - name: app
+    description: The value of the app label
+    default_value: ~
+source_url: "https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/"
```
Comment (alokedesai) [inline specs/kubernetes/list_pod_images_by_app.yaml:12]: Is there a better source of documentation we could link to here instead? This URL doesn't give much indication about what the command does.

File: specs/kubernetes/list_pod_images_by_app.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+name: List images by app label of pods
```
Comment (alokedesai) [inline specs/kubernetes/list_pod_images_by_app.yaml:2]: nit: could we make the name of this workflow more specific to the tool?

How about: 
"List all Kubernetes pods by app label"?

**Review by alokedesai — CHANGES_REQUESTED** [review]
Thanks for contributing! I have a few small comments, but besides that, it looks great

File: specs/kubernetes/list_pod_images_by_app.yaml (hunk 2 above)
Comment (pkaramol) [reply specs/kubernetes/list_pod_images_by_app.yaml]: Changed according to your suggestion.

File: specs/kubernetes/list_pod_images_by_app.yaml (hunk 1 above)
Comment (pkaramol) [reply specs/kubernetes/list_pod_images_by_app.yaml]: I switched to the url that documents how to use the `JSONPath`. Let me know if this is not relevant enough.

**Review by elviskahoro — APPROVED** [review]
Thanks for making the addressing the changes aloke recommended. Apologies for getting to this late.

if you're in our discord server I can add the appropriate badges just let me know what your discord username is

**Pull request merged**